fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/JobsServlet.java

```diff
@@ -164,8 +165,15 @@ public void getJobs(
                                     )
                     );
 
-            apiRequest.getJobViews()
-                    .toList()
+            Observable<Map<String, String>> jobViews;
+
+            if (filters == null || "".equals(filters)) {
+                jobViews = apiRequest.getJobViews();
```

Comment:
Sorry to be late bringing this up, but in the other ApiRequest objects we've avoided using the 'get' verb when the method is essentially a factory.  Since the ApiRequest continues to be both a data object and a factory, it would be completely obscure whether 'getFoo()' was retrieving a built foo or an accessor for an already built foo.  

This applies to getJobViews, but not by any means to just getJobViews in this code now.